Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -53,7 +53,7 @@ u16 GetEncodedSampleCount(u32 format)
 
 // block dimensions : widthStride, heightStride
 // texture dims : width, height, x offset, y offset
-static void WriteSwizzler(char*& p, u32 format, API_TYPE ApiType)
+static void WriteSwizzler(char*& p, u32 format, API_TYPE ApiType, int numInputSamples)
```

Comment:
I really don't like that i have to add a parameter everywhere... should I rather use a global? Or pack all args together  in a struct?